tox.ini

```diff
@@ -55,6 +56,14 @@ commands =
     mypy --python-version 3.5 src/build
     mypy --python-version 2.7 src/build
 
+[testenv:{py27, py35, py36, py37, py38, py39}-minversion]
```

Comment:
@gaborbernat is this the correct way to do this? (Also, I could add pypy and then combine the envlist  lines above, I think)